ios/manifest

```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 3.0.2
+version: 3.0.3
```

Comment:
Its a feature, so it needs a minor version bump.

Reply:
You are right. Updated to 3.1.0.